integration_tests/cases/env/cluster/ddl/alter_table.sql

```diff
@@ -18,3 +18,11 @@ DESCRIBE TABLE `05_alter_table_t0`;
 SELECT * FROM `05_alter_table_t0`;
 
 DROP TABLE `05_alter_table_t0`;
+
+-- alter table options
+CREATE TABLE `05_alter_table_t1`(a int, t timestamp NOT NULL, TIMESTAMP KEY(t)) ENGINE = Analytic;
+ALTER TABLE `05_alter_table_t1` MODIFY SETTING write_buffer_size='300M';
```

Comment:
Two suggestions:
1. After alter, you need to `show create table xxx` to ensure it work as expected.
2. You should drop `05_alter_table_t1` after use.